docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -137,8 +137,7 @@ When you import code, {ProductName} analyzes your repository to determine the ri
 +
 If {ProductName} finds a Dockerfile in your repository, it determines that a Dockerfile runtime is the most suitable for building and deploying you application. However, if the Dockerfile is not saved in one of the following expected locations, an error in determining the right runtime for your components can occur:
 +
-[#expected Dockerfile locations]
-.Expected Dockerfile locations
```

Comment:
I believe what you had earlier was fine. Since this was going to be a link, I believe we can make it one word or add "_". 

For example, [#expected_Dockerfile_locations]
Now where ever you want to add a link, use <<expected_Dockerfile_locations>>

Reply:
thank you! changed